pkg/service/retrieval/ucan_fx_test.go

```diff
@@ -132,6 +132,72 @@ func TestFXSpaceContentRetrieve(t *testing.T) {
 		assertContentRetrieveOK(t, inv.Link(), xres, hres, blob.bytes[:2], len(blob.bytes))
 	})
 
+	t.Run("space/content/retrieve without range", func(t *testing.T) {
+		space := testutil.RandomSigner(t)
+		randBytes := testutil.RandomBytes(t, 256)
+		blob := struct {
+			bytes []byte
+			cid   cid.Cid
+		}{randBytes, cid.NewCidV1(cid.Raw, testutil.MultihashFromBytes(t, randBytes))}
+
+		require.NoError(t, svc.Blobs().Allocations().Put(t.Context(), allocation.Allocation{
+			Blob: allocation.Blob{
+				Digest: blob.cid.Hash(),
+				Size:   uint64(len(blob.bytes)),
+			},
+			Space:   space.DID(),
+			Expires: 0,
+			Cause:   testutil.RandomCID(t),
+		}))
+		err := svc.Blobs().Store().Put(
+			t.Context(),
+			blob.cid.Hash(),
+			uint64(len(blob.bytes)),
+			bytes.NewReader(blob.bytes),
+		)
+		require.NoError(t, err)
+
+		prf := delegation.FromDelegation(
+			testutil.Must(
+				delegation.Delegate(
+					space,
+					testutil.Bob,
+					[]ucan.Capability[content.RetrieveCaveats]{
+						ucan.NewCapability(
+							content.RetrieveAbility,
+							space.DID().String(),
+							content.RetrieveCaveats{
+								Blob: content.BlobDigest{Digest: blob.cid.Hash()},
```

Comment:
Maybe this isn't a bug, and range is always required to be set? Issue/foot-gun is that when range isn't passed the defaults for `Start` and `End` are `0`, which is valid, but probably not what we want?